src/runtime/manifest-ast-types/manifest-ast-nodes.ts

```diff
@@ -800,21 +800,110 @@ export interface SchemaAlias extends BaseNode {
 
 export type Expression = ExpressionEntity;
 
+export type PaxelFunctionName = 'now' | 'min' | 'max' | 'average' | 'count' | 'union' | 'first';
```

Comment:
Tiny suggestion, I've found that if we make these as enums typescript will let us do set membership without having to maintain a separate list or do typelevel trickery.